src/lib/node.ts

```diff
@@ -33,7 +37,7 @@ export function nodeHmacSha256Sign(
 ): Uint8Array {
   const buf = crypto
     .createHmac(HMAC_NODE_ALGO, new Uint8Array(key))
```

Comment:
```suggestion
    .createHmac(HMAC_NODE_ALGO, Buffer.from(new Uint8Array(key)))
```

that worked on my end, maybe it should be the same for  all arrays defined like that